src/libstore/build/local-derivation-goal.cc

```diff
@@ -1517,7 +1517,7 @@ void LocalDerivationGoal::startDaemon()
                 try {
                     daemon::processConnection(store, from, to,
                         daemon::NotTrusted, daemon::Recursive,
-                        [&](Store & store) { store.createUser("nobody", 65535); });
+                        [&](Store & store) {});
```

Comment:
I think we can remove the `authHook` argument to `processConnection()` now, since it's only used to call `createUser()`.